test/specs/bounded.vcl

```diff
@@ -1,4 +1,4 @@
-network f : Real -> Real
+network f : Tensor Real [1] -> Tensor Real [1]
```

Comment:
Same for this unfortunately, its part of the test suite so changing it will cause tests to fail. Please could remove the changes from the PR?